packages/cxl-ui/src/components/cxl-like-or-dislike.js

```diff
@@ -0,0 +1,143 @@
+import { customElement, LitElement, html, property, query, queryAll } from 'lit-element';
+import '@conversionxl/cxl-lumo-styles';
+import cxlLikeOrDislikeStyles from '../styles/cxl-like-or-dislike-css.js';
+
+@customElement('cxl-like-or-dislike')
+export class CXLLikeOrDislikeElement extends LitElement {
+  @property({ type: Number })
+  upVotes = 0;
+
+  @property({ type: Number })
+  value = 0;
+
+  @property({ type: Number })
+  postId;
+
+  @property({ type: Number })
+  userId;
+
+  @property({ type: String })
+  postType;
+
+  @query('[counter]')
+  counter;
+
+  @query('.vote[up]')
+  voteUp;
+
+  @query('.vote[down]')
+  voteDown;
+
+  @queryAll('.vote')
+  voteAll;
+
+  /**
+   * API Url to which we make a POST request
+   *
+   * @type {string}
+   */
+  @property({ type: String })
+  apiUrl = 'https://jsonplaceholder.typicode.com/posts';
+
+  static get styles() {
+    return [cxlLikeOrDislikeStyles];
+  }
+
+  async _upVote(event) {
+    this.value = this.value === 1 ? 0 : 1;
```

Comment:
This seems to only toggle between 0 and 1.

Reply:
what is your suggestion here?

Reply:
Sorry, I may have jumped the gun. Let me spend some more time to understand everything :)

Reply:
I see what's happening now. It's tracking state for the unique user only.